src/_pytest/config/__init__.py

```diff
@@ -177,7 +177,7 @@ def get_config(args=None, plugins=None):
     config = Config(
         pluginmanager,
         invocation_params=Config.InvocationParams(
-            args=args or (), plugins=plugins, dir=Path().resolve()
+            args=args or (), plugins=plugins, dir=Path().absolute()
```

Comment:
based on the docstring of `Path.absolute` and the contents of `Path.resolve` i believe we need to normalize the absolute path to avoid regression/issues

also the issue is only partially shifted, as practically `tmp_path.resolve()` returns something unusable as  far as i understood - we should take a look at figuring when that happens and warn users that they are potentially in trouble

its perfectly fine to inform the user that pythons `pathlib` itself is broken and that pytest cant elevate that and just mitigates it enough to work sometimes

Reply:
@RonnyPfannschmidt 

The other use cases I found in pytest were related to `tmpdir` and `conftest`... in those cases, I think that resolving symlinks is ok (for `conftest` it uses it to create a cache key and for `tempdir`, it shouldn't matter that much where it's located).

The main problem (IMHO) is really on the current dir as the reported paths are usually printed relative to it.

Related to normalizing it, in this particular case, I don't think we need to do additional normalization as the only entry is the current dir and not a custom path (so, there should be nothing additional to normalize).

Reply:
then perhaps we should use `Path.cwd()` to begin with ?

Reply:
Yes, I agree, that's better than `Path.absolute`.

Reply:
Done